keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -47,7 +48,12 @@ class ResourceActuator(
     val id = resource.id
     val plugin = handlers.supporting(resource.apiVersion, resource.kind)
 
-    // todo eb: extract "application veto" as its own concept, and check before generating the diff.
+    if (resourcePauser.isPaused(resource)) {
+      log.debug("Actuation for resource {} is paused, skipping checks", id)
+      publisher.publishEvent(ResourceActuationPaused(resource, "Management of this resource has been paused"))
```

Comment:
Publishing an event here (which may be a repeat, but won't get stored if so) so that if an application is paused and then a resource is created it still gets the correct paused status.